stable/nextcloud/templates/deployment.yaml

```diff
@@ -34,6 +34,21 @@ spec:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        {{- if .Values.lifecycle }}
```

Comment:
Can you please also add documentation for the `lifecycle` value to the `README.md`?

Reply:
Added and updated the PR.